src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -67,6 +66,32 @@ public static Entity getEntityFromDatastore(String entityName, String entityProp
     return entity;
   }
 
+     /**
+    * This function takes in a list of cookies and matches a sessionID cookie
+    * with a specific name/value pair to a user entity with the same name/value
+    * pair as one of its properties. This user entity is then returned. 
+    * 
+    * <p>
+    * 
+    * @param  cookies    an array of cookies (usually all cookies on front end)
+    * @param  datastore  the database where entities are stored
+    * @return            a single entity that has the cookie name/value pair as a property
```

Comment:
Is this the same "user entity" that's used elsewhere in the code?

If not, might it be better to return a type other than an entity? E.g. the userId, which can then be used to look up the user? Entities seems very hard to work with -- many of the changes I've been requesting on PRs lately have been related to handling of Entities.

Reply:
this is the same user entity used elsewhere

Reply:
u can get the user ID and all the user's info from the entity that is retrieved here

Reply:
Okay cool. Please mention in the documentation that this is the User entity. Currently the doc implies that it's something different

Reply:
Actually nvm it's fine as is